htdocs/langs/sv_SE/main.lang

```diff
@@ -14,21 +14,21 @@ FONTFORPDF=helvetica
 FONTSIZEFORPDF=10
 SeparatorDecimal=,
 SeparatorThousand=Space
-FormatDateShort=%m/%d/%Y
-FormatDateShortInput=%m/%d/%Y
-FormatDateShortJava=MM/dd/yyyy
-FormatDateShortJavaInput=MM/dd/yyyy
-FormatDateShortJQuery=mm/dd/yy
-FormatDateShortJQueryInput=mm/dd/yy
-FormatHourShortJQuery=HH:MI
-FormatHourShort=%I:%M %p
+FormatDateShort=%Y-%m-%d
```

Comment:
This must be modified on transifex.

Reply:
@eldy yes but can you add me to all languages in transifex  please? Thanks